tests/methods/aci-18/input.dat

```diff
@@ -2,7 +2,7 @@ import forte
 
 e_scf = -39.72595399459846
 en_ref0 = -39.805891312415
-en_ref1 = -28.917284957659
+en_ref1 = -28.91728496215
 
```

Comment:
Please replace `#  basis cc-pcvdz` with `e_convergence 10` to ensure we converge the SCF procedure tightly.